### DISCUSSION TITLE
my github account was hacked

### DISCUSSION THREAD

**Discussion by amonASindy** [community/community#137409] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

the previous email was amonprograms@gmail.com  now it has been changed by hackers to haddenr855@gmail.com
help me My account has purchased I also subscribed to GitHub copilot monthly.

![image](https://github.com/user-attachments/assets/3d4fe377-f4a8-4cad-b2f5-3fb590b83e78)

**Comment by shinybrightstar** [comment]
Hi @Appletorsh508 ! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)
